src/App.js

```diff
@@ -1,36 +1,13 @@
 import React from 'react';
-
-const superHeros = [
-  {
-    id: 1,
-    name: "Wonderwoman",
-    description: "Can be invisible"
-  },
-  {
-    id: 2,
-    name: "Spiderman",
-    description: "Can climb walls"
-  }
-]
-
-
-// function App() {
-//   return (
-//     <div className="App">
-//       {superHeros.map(hero => {
-//          return <h1>{hero.name}</h1>
-//       })}
-//     </div>
-//   );
-// }
+import Superheros from './Superheros'
```

Comment:
Usually we capitalize component names. Collections should start with lowercase letter. So here we should have `superheroes` instead. Components starting with capital letter is a convention we use, so if I was working on this file and Superheroes (with capital S) where coming from a package or library I would try to render them like this `<Superheroes />`

Reply:
Have amended this to the correct casing and have added some inline style to the JSX :)